mkdocs.yml

```diff
@@ -52,6 +52,8 @@ nav:
         - Level Editing:
           - Sonic GLvl: guides/hedgehog-engine/blueblur/levels/glvl.md
       - Sonic Frontiers:
+       - Cyloop Color:
+          - Replacing the Color: guides/hedgehog-engine/rangers/Cyloop/color.md
```

Comment:
Try to write paths in lower case only. So in this case, replace "Cyloop" with "cyloop"